cmd/uhura/main.go

```diff
@@ -0,0 +1,248 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/Symantec/Dominator/lib/logbuf"
+	"github.com/Symantec/scotty/lib/apiutil"
+	"github.com/Symantec/scotty/lib/dynconfig"
+	"github.com/Symantec/scotty/lib/yamlutil"
+	"github.com/Symantec/scotty/tsdbjson"
+	"github.com/Symantec/tricorder/go/healthserver"
+	"github.com/Symantec/tricorder/go/tricorder"
+	"github.com/Symantec/tricorder/go/tricorder/units"
+	"github.com/Symantec/uhura/chreader"
+	"github.com/Symantec/uhura/cmd/uhura/splash"
+	"github.com/Symantec/uhura/tsdbadapter"
+	"io"
+	"log"
+	"net/http"
+	"net/rpc"
+	"net/url"
+	"path"
+	"time"
+)
+
+var (
+	fPort      = flag.Int("port", 4242, "port")
```

Comment:
How about using "portNum" as the flag name? That would make it consistent with **Dominator** daemons.
Also, how about picking a port in the 69xx range? Maybe 691x?